rero_ils/modules/loans/utils.py

```diff
@@ -166,11 +167,17 @@ def can_be_requested(loan):
 
     if not loan.item_pid:
         raise Exception('Transaction on document is not implemented.')
-    # 1) Check if circulation_policy allows request
+
+    # 1) Check if there is already a loan for same patron+item
+    if get_any_loans_by_item_pid_by_patron_pid(
+            loan.get('item_pid').get('value'), loan.get('patron_pid')):
```

Comment:
If the `item_pid` required for a loan, I would do `loan['item_pid']` else `loan.get('item_pid', {})`

Reply:
The indentation is weird.